packages/web/src/pages/sign-up-page/pages/CreateEmailPage.tsx

```diff
@@ -0,0 +1,223 @@
+      dispatch(setLinkedSocialOnFirstPage(true))
+      dispatch(setValueField('handle', handle))
+      navigate(
+        requiresReview ? SIGN_UP_HANDLE_PAGE : SIGN_UP_CREATE_LOGIN_DETAILS
+      )
+    },
+    [dispatch, navigate]
+  )
+
+  const onSubmit = useCallback(
+    async (
+      values: SignUpEmailValues,
+      { setErrors }: FormikHelpers<SignUpEmailValues>
+    ) => {
+      const { email } = values
+      if (queryContext !== null) {
+        try {
+          // Check identity API for existing emails
+          const emailExists = await signUpFetch.isEmailInUse(
+            { email },
+            queryContext
+          )
+          // Set the email in the store
+          dispatch(setValueField('email', values.email))
+          if (emailExists) {
+            // Redirect to sign in if the email exists already
+            navigate(SIGN_IN_PAGE)
+          } else {
+            // Move onto the password page
+            navigate(SIGN_UP_PASSWORD_PAGE)
+          }
+        } catch (e) {
+          // Unknown error state ¯\_(ツ)_/¯
+          setErrors({ email: messages.unknownError })
+        }
+      }
+    },
+    [dispatch, navigate, queryContext]
+  )
+
+  return (
+    <Formik
+      validationSchema={FormSchema}
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+      validateOnBlur
+      validateOnChange={false}
+    >
+      {({ isSubmitting }) => (
+        <Flex
+          as={Form}
+          direction='column'
+          h='100%'
+          w='100%'
+          ph={isMobile ? 'l' : '2xl'}
+          pv='2xl'
+          gap='2xl'
+        >
+          <Box alignSelf='center'>
+            {isMobile ? (
+              <IconAudiusLogoHorizontalColor />
+            ) : (
+              <PreloadImage
+                src={audiusLogoColored}
+                alt='Audius Colored Logo'
+                css={{
+                  height: 160,
+                  width: 160,
+                  objectFit: 'contain'
+                }}
+              />
+            )}
+          </Box>
+          <Flex direction='column' gap={isMobile ? 's' : 'l'}>
+            <Text
+              variant='heading'
+              size={isMobile ? 'm' : 'l'}
+              color='accent'
+              tag='h1'
+              css={{ textAlign: isMobile ? 'center' : undefined }}
+            >
+              {messages.title}
+            </Text>
+            <Text
+              color='default'
+              size={isMobile ? 'm' : 'l'}
+              variant='body'
+              tag='h2'
+              css={{ textAlign: isMobile ? 'center' : undefined }}
+            >
+              {messages.subHeader}
+            </Text>
+          </Flex>
+          <Flex direction='column' gap='l'>
+            <HarmonyTextField
+              name='email'
+              autoComplete='email'
+              label={messages.emailLabel}
+            />
+            <Divider css={{ width: '100%' }}>
```

Comment:
feels like this should be built-in to Divider?

Reply:
wasnt needed in fact